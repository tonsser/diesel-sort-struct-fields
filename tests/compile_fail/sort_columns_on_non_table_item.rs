#![allow(unused_imports)]

#[macro_use]
extern crate diesel;

use diesel::prelude::*;
use diesel_sort_struct_fields::sort_columns;

#[sort_columns]
fn main() {}
