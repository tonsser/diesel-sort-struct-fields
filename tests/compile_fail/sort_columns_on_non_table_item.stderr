error: `#[sort_columns]` only works on the `diesel::table!` macro
  --> $DIR/sort_columns_on_non_table_item.rs:10:1
   |
10 | fn main() {}
   | ^^

error[E0601]: `main` function not found in crate `$CRATE`
  |
  = note: consider adding a `main` function to `$DIR/tests/compile_fail/sort_columns_on_non_table_item.rs`

For more information about this error, try `rustc --explain E0601`.
